build.gradle

```diff
@@ -810,29 +840,59 @@ def exampleJavaEEContent = project.copySpec {
 }
 
 /** binary distribution in zip format. */
-task distributeBinaryZip(type: Zip, group: 'distribution', description: 'Distributes binary archive.', dependsOn: [mainJar, emfJar, aspectJJar, sourcesJar, javadocJar,
-  'kieker-examples:buildStandaloneExamples',
-  ':kieker-tools:behavior-analysis:distZip',
-  ':kieker-tools:collector:distZip',
-  ':kieker-tools:convert-logging-timestamp:distZip',
-  ':kieker-tools:log-replayer:distZip',
-  ':kieker-tools:resource-monitor:distZip',
-  ':kieker-tools:trace-analysis:distZip',
-  ':kieker-tools:trace-analysis-gui:distZip',
-  ':kieker-examples:jar',
-  ':kieker-examples:spotlessMisc' ]) {
-  archiveClassifier = 'binaries'
-  with binaryContent, toolsContent, libraryContent, exampleContent, exampleJavaEEContent
-}
+//task distributeBinaryZip(type: Zip, group: 'distribution', description: 'Distributes binary archive.', dependsOn: [mainJar, emfJar, aspectJJar, sourcesJar, javadocJar,
```

Comment:
Why is this commented code left in the source?

Reply:
As we did not finally discuss to never ever support a zip based binary distribution.